azure-webapp-maven-plugin/pom.xml

```diff
@@ -317,30 +281,6 @@
                     </execution>
                 </executions>
             </plugin>
-            <plugin>
```

Comment:
https://github.com/microsoft/azure-maven-plugins/pull/81, this is for better development experience in Eclipse, do we need to remove it?

Reply:
I will restore it.